codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -272,6 +275,13 @@ private boolean readClientPrefaceString(ByteBuf in) throws Http2Exception {
             if (bytesRead == 0 || !ByteBufUtil.equals(in, in.readerIndex(),
                                                       clientPrefaceString, clientPrefaceString.readerIndex(),
                                                       bytesRead)) {
+                String http1Request =
```

Comment:
> "HTTP/1.1" is the last part of the first line

👍  ... 🤦‍♂️ 

> ByteBuf lacks a "contains" method

we could add a method to `ByteBufUtil` which borrows from [AsciiString#indexOf](https://github.com/netty/netty/blob/4.1/common/src/main/java/io/netty/util/AsciiString.java#L675) (which is what `AsciiString#contains` uses).